ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -772,3 +772,38 @@ literals:
 - zone_air_occupied_heating_temperature_setpoint
 - mixed_air_relative_humidity_sensor
 - mixed_air_dewpoint_temperature_sensor
+
+- heating_supply_water_temperature_sensor
+- heating_return_water_temperature_sensor
+- heating_thermal_power_sensor
+- heating_thermal_energy_accumulator
+
+#added for UK-LON-X1
+
+- heating_wheel_percentage_sensor
+- heat_recovery_damper_percentage_sensor
+- discharge_air_filter_differential_pressure_sensor
+- discharge_air_filter_differential_pressure_setpoint
+- return_air_filter_differential_pressure_sensor
+- return_air_filter_differential_pressure_setpoint
+- cooling_coil_percentage_command
+- cooling_coil_status:
+  - ACTIVE
+  - INACTIVE
+- heating_coil_percentage_command
```

Comment:
why are you using this and not heating_water_valve_percentage command? is it not hydronic? if not, what is it?